src/anbox/cmds/check_features.cpp

```diff
@@ -30,22 +30,22 @@ std::vector<std::string> cpu_whitelist = {
   // The following CPUs do not support AVX and without it cpu_features can't detect
   // if SSE & friends are supported. See https://github.com/google/cpu_features/issues/4
 
+  // Intel Core i7 Q720
+  "Q 720",
   // Intel Core i7 M620
   "M 620",
   // Intel Core i5 M460
   "M 460",
   // Intel Celeron N2840
   "N2840",
-  // Intel Core i7 Q720
-  "Q 720",
-  // Intel Pentium T4500
-  "T4500", 
-  // Intel Core i7 Q720
-  "Q 720",
   // Intel Xeon E5520
-  "E5520"
+  "E5520",
+  // Intel Pentium T4500
+  "T4500",
   // Intel Core2 Duo T6500
-  "T6500"
```

Comment:
Actually we have to drop this one too as it doesn't support SSE 4.2. See http://www.cpu-world.com/CPUs/Core_2/Intel-Core%202%20Duo%20Mobile%20T6500%20AW80577GG0452MA%20-%20AW80577GG0452ML.html

Reply:
So SSE 4.1 is no longer viable to run anbox?

Reply:
No, as stated in the code you need SSE 4.2 too. Did you run Anbox before with just SSE 4.1?

Reply:
I was never able to get it to install. I thought at some point I had seen a thread or documentation stating that SSSE 3 and SSE 4.1 were supported, but if that is not correct than I do apologize. I was just hoping that whitelisting my CPU was all that was needed to get me up and running.

Reply:
Nope, whitelisting wont help as long as the CPU doesn't support the necessary instructions.